dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java

```diff
@@ -26,22 +26,22 @@ public class HeartBeat {
 
     private static final Logger logger = LoggerFactory.getLogger(HeartBeat.class);
 
-    private long startupTime;
-    private long reportTime;
-    private double cpuUsage;
-    private double memoryUsage;
-    private double loadAverage;
-    private double availablePhysicalMemorySize;
-    private double maxCpuloadAvg;
-    private double reservedMemory;
-    private int serverStatus;
-    private int processId;
-
-    private int workerHostWeight; // worker host weight
-    private int workerWaitingTaskCount; // worker waiting task count
-    private int workerExecThreadCount; // worker thread pool thread count
-
-    private double diskAvailable;
+    protected long startupTime;
+    protected long reportTime;
+    protected double cpuUsage;
+    protected double memoryUsage;
+    protected double loadAverage;
+    protected double availablePhysicalMemorySize;
+    protected double maxCpuloadAvg;
+    protected double reservedMemory;
+    protected int serverStatus;
+    protected int processId;
+
+    protected int workerHostWeight; // worker host weight
+    protected int workerWaitingTaskCount; // worker waiting task count
+    protected int workerExecThreadCount; // worker thread pool thread count
```

Comment:
These fields need to move to `WorkerHeartBeat`?

Reply:
I think these attributes should be a part of  `HeartBeat`, but only apply to `WorkerHeartBeat`. 
The advantage is that the processing of `HeartBeat` is consistent， like `decodeHeartBeat`

Reply:
It's better to move these field to subclass, and maybe we can use directly use Json to serialize/deserialize the heartbeat? cc @caishunfeng WDYT

Reply:
there are pros and cons, anything will do